packages/recaps/src/index.ts

```diff
@@ -0,0 +1,93 @@
+type CeramicWrite = {
+  "ceramic/write": []
+}
```

Comment:
Do we want to keep Ceramic specific things outside of here? Currently all Ceramic specific code of SIWx lives in the js-ceramic codebase.

Reply:
Mostly likely in the end, but thinking that for now everything ceramic specific goes here for now (to keep more self contained while experimental), and more useful to drive with specific use case first (ceramic), then generalize after 

even if specs dont have namespaces, a general library will need to have some form of that, similar to spruce libs, lots of the rules/parsing are defined by that 

likely will make sense to combine most of siwx + recap things together instead this separate library here